src/components/portrait/portrait.style.tsx

```diff
@@ -1,157 +1,78 @@
 import React from "react";
-import styled, { css } from "styled-components";
+import styled from "styled-components";
 import { margin, MarginProps } from "styled-system";
 
 import BaseTheme from "../../style/themes/base";
 import Icon, { IconType } from "../icon";
 import { PortraitSizes, PortraitShapes } from "./portrait.component";
 import { PORTRAIT_SIZE_PARAMS } from "./portrait.config";
+import profileConfigSizes from "../profile/profile.config";
 
-type StylingForSize = {
+type StyledPortraitProps = {
+  darkBackground?: boolean;
   size: PortraitSizes;
-  initials?: string;
-};
-
-function stylingForSize({ size, initials }: StylingForSize) {
-  const params = PORTRAIT_SIZE_PARAMS[size];
-
-  if (initials) {
-    return css`
-      width: inherit;
-      height: inherit;
-      margin: 1px;
-    `;
-  }
-
-  return css`
-    width: ${params?.dimensions}px;
-    height: ${params?.dimensions}px;
-  `;
-}
-
-interface StylingForShape {
   shape?: PortraitShapes;
-}
-
-function stylingForShape({ shape }: StylingForShape) {
-  let cssString = "overflow: hidden;";
-
-  if (shape === "square") cssString += "border-radius: 0px;";
-  if (shape === "circle")
-    cssString += "border-radius: var(--borderRadiusCircle);";
-
-  return css`
-    ${cssString}
-  `;
-}
-
-type StylingForIcon = {
-  size: PortraitSizes;
-  darkBackground: boolean;
-};
-
-function stylingForIcon({ size, darkBackground }: StylingForIcon) {
-  const params = PORTRAIT_SIZE_PARAMS[size];
-
-  let color = "var(--colorsUtilityMajor200)";
-  let backgroundColor = "var(--colorsUtilityMajor025)";
-
-  if (darkBackground) {
-    color = "var(--colorsUtilityMajor025)";
-    backgroundColor = "var(--colorsUtilityMajor200)";
-  }
-
-  return css`
-    background-color: ${backgroundColor};
-    color: ${color};
-    ::before {
-      font-size: ${params?.iconDimensions}px;
-    }
-  `;
-}
-
-export function getColorsForInitials(
-  darkBackground?: boolean
-): {
-  textColor: "colorsUtilityYang100" | "colorsUtilityYin055";
-  bgColor: "colorsUtilityMajor400" | "colorsUtilityMajor025";
-} {
-  return {
-    textColor: darkBackground ? "colorsUtilityYang100" : "colorsUtilityYin055",
-    bgColor: darkBackground ? "colorsUtilityMajor400" : "colorsUtilityMajor025",
-  };
-}
-
-type PortraitSizeAndShape = {
-  size: PortraitSizes;
-  shape?: PortraitShapes;
-};
-
-type StyledPortraitInitialsProps = PortraitSizeAndShape & {
-  initials?: string;
+  type: IconType;
```

Comment:
Question: Is this used anywhere? I think you get this for free by using `styled(Icon)`